GETTING_STARTED.md

```diff
@@ -0,0 +1,63 @@
+# Getting Started as a Boutiques Developer
+
+This guide will help you set up your local environment for coding and testing Boutiques.
+
+## 0. Things you'll need
+- **A Linux system**. It _may_ be possible to develop Boutiques on Windows, but will likely cause many headaches.
+- **Root (sudo) access**. Unless your system already has everything you need installed, you'll need root permission to install new programs.
+- **Python**. Boutiques supports any version >= 2.7. It's a good idea to have multiple versions of Python installed (e.g. 2.7 and 3.7) so you can test your code for version compatibility.
+- **pip**. Most versions of Python will come with it, but in case you don't have it, instructions to install it are [here](https://pip.pypa.io/en/stable/installing/).
+- **Docker**. If you don't already have Docker, follow the [instructions](https://docs.docker.com/install/overview/) to install Docker CE.
+- **Singularity**. Follow the instructions [here](https://singularity.lbl.gov/install-linux) to download the latest stable release of Singularity.

```

Comment:
Yeah, maybe say "Docker or Singularity" here, because it depends greatly on the system you're using and _generally_ if it works for one it'll work for both, unless you're editing that specific section of code.